### ISSUE TITLE
Font references cached across frames

### ISSUE THREAD

**Issue by cschol** [alexey-komarov/fruitsofkarma-vcvrack#10]
It looks like font references are cached across frames, which causes the Rack VST to crash when using this plugin.

The offending code looks to be here:

```
src/ScaleMerger.cpp
613-            };
614-
615-            {
616:                    std::shared_ptr<Font> fnt = APP->window->loadFont(
617-                            asset::plugin(pluginInstance, "res/fonts/DejaVuSansMono.ttf")
618-                    );
619-
```

See [migration manual](https://vcvrack.com/manual/Migrate2#2-1-Don-t-store-Font-and-Image-references-across-multiple-frames) for more information on how to fix this.

**Comment by alexey-komarov** [comment]
@cschol 

I have read the doc you provided, the required fix is clear to me, thank you
Could you please kindly check this PR: https://github.com/alexey-komarov/fruitsofkarma-vcvrack/pull/11/files

Also, I am not sure about version 1.0.

I am loading **std::shared_ptr<Font>**  outside of TransparentWidget and pass the same instance to the TransparentWidget constructor for each of two widgets. Is that ok or it's better to load font separately in the constructor?

**Comment by cschol** [comment]
Yes. Loading the font in `drawLayer` prevents the caching and is the correct way to handle this.

We are not building or distributing any updated 1.0 versions anymore. The issue only affects the VST version, which is only available for Rack 2.0.

**Comment by alexey-komarov** [comment]
Hi school,
I found a minor bug, could you please rebuild the plugin, the latest commit for v2 is here: https://github.com/alexey-komarov/fruitsofkarma-vcvrack/commit/083f6a12d09e6b3764c83f2b62fcfa3379460475

You also mentioned you don't build v1 anymore, but does it mean you also don't build plugins for v1, too? I.E. does it make sense to provide a link to the fixed version of my plugin for VCVRack v1?
